### ISSUE TITLE
Feature suggestion: Ability to hide HN advertising aka "job ads"

### ISSUE THREAD

**Issue by nick-s-b** [plibither8/refined-hacker-news#32]
Hi,
HN is often full of YC companies' job ads since they get a free boost on the front page. And the only thing you can do is hide them.
I wish there was a way to hide them automatically. Can someone make it happen?
Thanks for an amazing extension ❤️ !

**Comment by plibither8** [comment]
Hey! When you say "hide", do you mean just removing them/making them invisible, or the HN button "hide"?

**Comment by nick-s-b** [comment]
@plibither8 Yes... making it equivalent to hitting hide. I just don't want to see them at all since those are useless to me.

**Comment by plibither8** [comment]
Such a feature, if implemented, will be better if disabled by default since not all users would want HN Jobs to be hidden. What do you say?

**Comment by nick-s-b** [comment]
@plibither8 Sure! An option that's not enabled by default is fine by me :)

**Comment by plibither8** [comment]
Alright then! I'll work on this for the next update :)

Cheers
